src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -236,11 +237,12 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
     }
 
     groupExercisesByType() {
-        const exercises = this.course!.exercises;
+        let exercises = this.course!.exercises;
         const groupedExercises: any[] = [];
         const exerciseTypes: string[] = [];
+        exercises = exercises.sort((a: Exercise, b: Exercise) => (a.dueDate || moment().add(5, 'year')).valueOf() - (b.dueDate || moment().add(5, 'year')).valueOf());
```

Comment:
The other thing I wonder is here why + 5 years? This seems arbitrary to me, especially because Artemis is not that old.
If this is as intended, a comment might help.

Reply:
i added a comment